test/integration/api/services_controller_test.rb

```diff
@@ -178,6 +207,19 @@ class SettingsTest < self
       assert_equal 'http://api.staging.example.com:8080', proxy.sandbox_endpoint
     end
 
+    test 'update authentication option' do
+      get settings_admin_service_path(service)
+
+      backend_version = service.backend_version
+      assert_equal '1', backend_version
+
+      backend_version = '2'
+
+      put admin_service_path(service), params: update_params
+      assert_equal 'Product information updated.', flash[:notice]
+      assert_equal '2', backend_version
```

Comment:
This is checking that `backend_version` equals `2` but we set this on line 216. What is the intended purpose of this assertion?

Reply:
I understand that it may seem redundant but I want to confirm it doesn't go back to the default option, since that was also part of the issue.

Reply:
But nothing can change this local variable `backend_version`. No expression between setting it to `2` and assetion can change it. Perhaps you want to check the value from the response? Or reload from database?

Reply:
Yep, that sounds right... 😞 I'll fix it so I can check the value from the response.